### PR TITLE
fix(UI): Fix path for centreon frontend

### PR DESCRIPTION
**Pull request by Jordan91**
## Description

Wrong path for centreon frontend

Sans titre

**Fixes** # (issue)

Fix path for centreon-frontend

## Type of change

- [x] Patch fixing an issue (non-breaking change)

## Target serie

- [x] 22.04.x (master)

### CODE REVIEW

**Review by Thebarda — DISMISSED** [review]

File: www/front_src/src/Provider/decoder.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { User } from '@centreon/ui-context';
 export const userDecoder = JsonDecoder.object<User>(
   {
     alias: JsonDecoder.string,
+    default_page: JsonDecoder.string,
```
[the remark this answers is not in the document]
Comment (Jordan91) [reply www/front_src/src/Provider/decoder.ts:8]: Thank's Tom

File: package.json [hunk 2]
```diff
@@ -93,7 +93,7 @@
     "webpack-merge": "^5.8.0"
   },
   "dependencies": {
-    "@centreon/centreon-frontend": "git+https://centreon@github.com/centreon/centreon-frontend.git#develop",
+    "centreon-frontend": "git+https://centreon@github.com/centreon/centreon-frontend.git#Fix-path-for-Jest",
```
Comment (Thebarda) [inline package.json:96]: Don't forget to re-use the `develop` branch

Comment (Jordan91) [reply package.json:96]: ^^ yes

File: www/front_src/src/Provider/index.tsx [hunk 3]
```diff
@@ -124,6 +124,7 @@ const AppProvider = ({ children }: Props): JSX.Element => {
         ]) => {
           setUser({
             alias: retrievedUser.alias,
+            // default_page: null,
```
Comment (bdauria) [inline www/front_src/src/Provider/index.tsx:127]:
```suggestion
            default_page: null,
```

**Review by bdauria — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**